connectivity/lorawan/tests/TESTS/lorawan/loraradio/main.cpp

```diff
@@ -64,39 +55,34 @@ static volatile event_t received_event;
 
 static void tx_done()
 {
-    rtos::ThisThread::sleep_for(2);
     TEST_ASSERT_EQUAL(EV_NONE, received_event);
     received_event = EV_TX_DONE;
     TEST_ASSERT_EQUAL(osOK, event_sem.release());
 }
 
 static void tx_timeout()
 {
-    rtos::ThisThread::sleep_for(2);
     TEST_ASSERT_EQUAL(EV_NONE, received_event);
     received_event = EV_TX_TIMEOUT;
     TEST_ASSERT_EQUAL(osOK, event_sem.release());
 }
 
 static void rx_done(const uint8_t *payload, uint16_t size, int16_t rssi, int8_t snr)
 {
-    rtos::ThisThread::sleep_for(2);
```

Comment:
Do you know why these sleep where present in the first place ?

Reply:
No, I am quite new in lora....